src/hooks/useOutputMaterial.ts

```diff
@@ -1,11 +1,18 @@
-import { IGeometryData, IMaterialAbstractData, IMaterialStandardDataProperties, IOutputApi, ITreeNode, MaterialStandardData } from "@shapediver/viewer";
+import { IGeometryData, IMaterialGemDataProperties, IMaterialSpecularGlossinessDataProperties, IMaterialStandardDataProperties, IMaterialUnlitDataProperties, IOutputApi, ITreeNode, MaterialGemData, MaterialSpecularGlossinessData, MaterialStandardData, MaterialUnlitData } from "@shapediver/viewer";
 import { useCallback, useEffect, useRef, useState } from "react";
 import { useOutputNode } from "./useOutputNode";
 
 
 const isGeometryData = (data: any): data is IGeometryData =>
 	"mode" in data && "primitive" in data && "material" in data;
```

Comment:
@MajorMeerkatThe3rd let's add some documentation with hints to the developer user to `isGeometryData` and `getGeometryData`

Reply:
@snabela Is there a reason why we do it like this and not with `instanceof`?
Because if there is a reason, I would just add a `isGeometryData` property, because this is currently not a very nice solution